microverstrail.css

```diff
@@ -0,0 +1,13 @@
+#navbar {
+    position: fixed;
+    top: 0;
+  }
+  
+  #welcome-section {
+    height: 100vh
+  }

```

Comment:
Please, make sure you read this [html and css best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
I highly recommend you to start using class attribute and only use id when is necessary.  :smile: :smile: :smile:

Reply:
Good 👍 job